packages/nx-plugin/src/generators/service/typed-json-config/tsconfig.json.ts

```diff
@@ -0,0 +1,42 @@
+import { joinPathFragments } from '@nrwl/devkit';
+
+import { NormalizedSchema, TsConfig } from '../../types';
+
+export const packageTsConfig = (options: NormalizedSchema): TsConfig => ({
+  extends: joinPathFragments(options.offsetFromRoot, 'tsconfig.json'),
+  compilerOptions: {
+    preserveSymlinks: true,
+    baseUrl: '.',
+    esModuleInterop: true,
+  },
+  references: [
+    {
+      path: joinPathFragments(
+        options.offsetFromRoot,
+        'contracts/core-contracts/tsconfig.build.json',
```

Comment:
This is probably not useful and may break if the monorepo does not have such package

Reply:
In the `serverless.ts` generated, we use the `httpApiRessourceContract` from `core-contracts` to get the `httpApi` id
So if this package doesn't exist, the generated service will also be broken

Reply:
That is correct, I'm wondering if this is a good idea though... Food for thought :thinking:

Reply:
I totally agree but for now I don't really see an alternative
we should talk about it next Friday 🙂